### ISSUE TITLE
Minor: Confusingly mentioning `SSH` in the output when connector is set to `winrm`: Waiting for SSH to become available...

### ISSUE THREAD

**Issue by uvsmtid** [vagrant-libvirt/vagrant-libvirt#635]
The original report from upstream:
https://github.com/mitchellh/vagrant/issues/7641

[It is said](https://github.com/mitchellh/vagrant/issues/7641#issuecomment-237024968) to be a bug of `vagrant-libvirt` which does not differentiate between WinRM and SSH:
https://github.com/vagrant-libvirt/vagrant-libvirt/blob/a963ff4c70f5e6ce5bc2589b7e98c1c0671e2ccc/locales/en.yml

---

This is a copy-and-paste of the original report:
### Vagrant version

Vagrant 1.8.5
### Host operating system

Fedora 24 x86_64
### Guest operating system

Windows Server 2012 R2
### Vagrantfile

This `Vagrantfile` can be used:
https://github.com/uvsmtid/vagrant-boxes/blob/develop/windows-server-2012-R2-gui/Vagrantfile
The Vagrant box is not published, however, it is not that relevant because it is reproducible with any `Vagrantfile` where `winrm` is used instead of `ssh`.
### Debug output

```
...
 INFO interface: info: Waiting for SSH to become available...
 INFO interface: info: ==> win_serv_2012R2_minion: Waiting for SSH to become available...
 INFO winrm: Checking whether WinRM is ready...
DEBUG provider: Searching for cap: winrm_info
DEBUG provider: Checking in: libvirt
DEBUG ssh: Checking key permissions: /home/uvsmtid/.vagrant.d/insecure_private_key
DEBUG winrmshell: initializing WinRMShell
 INFO winrmshell: Attempting to connect to WinRM...
 INFO winrmshell:   - Host: 192.168.121.253
 INFO winrmshell:   - Port: 5985
 INFO winrmshell:   - Username: Administrator
 INFO winrmshell:   - Transport: negotiate
DEBUG winrmshell: [WinRM] opening remote shell on http://192.168.121.253:5985/wsman
DEBUG winrmshell: [WinRM] remote shell E61B7680-AE95-40EC-8B41-199ACEB8FB4A is open on http://192.168.121.253:5985/wsman
DEBUG winrmshell: Output: {:data=>[{:stdout=>"WIN-O0603JH5D6C"}, {:stdout=>"\r\n"}], :exitcode=>0}
 INFO winrm: WinRM is ready!
...
```
### Expected behavior

It should mention `WinRM` if it is configured instead of `SSH`:

```
 INFO interface: info: Waiting for WinRM to become available...
```
### Actual behavior

It mentions `SSH`:

```
 INFO interface: info: Waiting for SSH to become available...
```
### Steps to reproduce

Just run `vagrant up` with `winrm` configured as communicator for Windows box.
### References

The confusing output always requires more steps to rule out case like misconfiguration of `winrm` (when it is actually fine). For example, only debug output in issue #4600 shows that Vagrant actually uses `winrm` internally while it still prints `SSH` by default.

**Comment by infernix** [comment]
Looks like we have to add something like `if config.vm.communicator != :winrm` to `wait_till_up.rb` then.

**Comment by electrofelix** [comment]
I think this should be solved by some of the fixes for #1292 which switches to vagrant's built-in WaitForCommunicator, which I'm assuming should display the correct information around the communicator.

I don't have any Windows VMs around to confirm this explicitly, but since we no wait for SSH explicitly in WaitTillUp and hand of to WaitForCommunicator, I'm going to make an assumption that this is resolved by that PR.
